lib/src/kraken.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:cln_plugin_api/cln_plugin.dart';
+
+class KrakenPlugin extends Plugin {
+  /// This is the Kraken doctor RPC method. This method allows for
+  /// analyzing the failure of invoice payments.
+  Future<Map<String, Object>> krakenDoctor(
+      Plugin plugin, Map<String, Object> request) {
+    log(level: "info", message: "This is the kraken doctor output.");
+    return Future.value({
+      "output": "Kraken doctor output~",
+    });
+  }
+
+  /// This is the Kraken pay RPC method. This method allows for
+  /// an alternate method to pay invoices in Core Lightning and
+  /// return payment failure analysis reports.
+  Future<Map<String, Object>> krakenPay(
+      Plugin plugin, Map<String, Object> request) {
+    log(level: "info", message: "This is the kraken pay output.");
+    return Future.value({
+      "greeting": "Kraken pay output!",
+    });
+  }
+
+  @override
+  void configurePlugin() {
+    registerRPCMethod(
+        name: "kraken_doctor",
+        usage: "",
+        description:
+            "Statically analyze a payment and provide insights in the case of a failure.",
```

Comment:
Suggestions @vincenzopalazzo ?

Reply:
suggestion about what?

Reply:
I reworded from the earlier comment, Is this a better description?

Reply:
Yeah! do not worry about it too much, I just make a suggestion because the suggestion was too generic and I do not understand if you had some part of the problem in the dark :)